report.py

```diff
@@ -210,6 +211,9 @@ def accum_timestats(self, message):
             tz_offset = config.default_tz_offset
         tz = user.get("tz", "Unknown")
         self.increment(["timezone", tz], message)
+        if tz not in self._data['posters_per_timezone']:
+            self._data['posters_per_timezone'][tz] = {}
+        self._data['posters_per_timezone'][tz][uid] = 1
```

Comment:
You could use collections.defaultdict instead.

Reply:
Yup. Done.